---
title: 
description: "Shaul Pollak Lab CMESS DOME University of Vienna Publications"
layout: "simple"
---

---

<b>Google Scholar profile: <a href="https://scholar.google.com/citations?user=jIHJECUAAAAJ&hl=en&oi=ao">Link</a></b>

---

## Preprints
Anna Lopatina, Mariusz Ferenc, Nina Bartlau, Michael Wolfram, Kerrin Steensen, Sebastiano Muscia, Fatima Hussain, Madeleine Schnurer, Leila Afjehi-Sadat, **Shaul Pollak**, Martin F Polz\
[Interpretable machine learning reveals a diverse arsenal of anti-defenses in bacterial viruses](https://www.biorxiv.org/content/10.1101/2024.06.14.598830.abstract)\
<u>bioRxiv</u>. 2024 Jun.

Xiaoqian Annie Yu, Cameron R Strachan, Craig W Herbold, Michaela Lang, Christoph Gasche, Athanasios Makristathis, Nicola Segata, **Shaul Pollak**, Adrian Tett, Martin F Polz\
[Genome-wide sweeps create fundamental ecological units in the human gut microbiome](https://www.biorxiv.org/content/10.1101/2024.05.25.595854v1.abstract)\
<u>bioRxiv</u>. 2024 May.


## Published
### 2024
Jamie W Becker, **Shaul Pollak**, Jessie W Berta-Thompson, Kevin W Becker, Rogier Braakman, Keven D Dooley, Thomas Hackl, Allison Coe, Aldo Arellano, Kristen N LeGault, Paul M Berube, Steven J Biller, Andrés Cubillos-Ruiz, Benjamin AS Van Mooy, Sallie W Chisholm
[Novel isolates expand the physiological diversity of Prochlorococcus and illuminate its macroevolution](https://journals.asm.org/doi/abs/10.1128/mbio.03497-23)
<u>Mbio</u>. 2024 Nov.

Ksenia Guseva, Moritz Mohrlok, Lauren Alteio, Hannes Schmidt, **Shaul Pollak**, Christina Kaiser\
[Bacteria face trade-offs in the decomposition of complex biopolymers](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012320)\
<u>PLOS Computational Biology</u>. 2024 Aug.

Xiaoyu Shan, Philip A Wasson, YuanQiao Rao, Scott Backer, Lyndsay Leal, Vurtice C Albright III, Wei Gao, Yunzhou Chai, Andreas Sichert, **Shaul Pollak**, Ligeng Yin, Otto X Cordero\
[Simple genomic traits predict rates of polysaccharide biodegradation](https://pubs.acs.org/doi/abs/10.1021/acs.est.4c02769)\
<u>Environmental Science & Technology</u>. 2024 Jul.

### 2018-2023
Bareia T., **Pollak S.**, Guler P., Puyesky S., Eldar A.\
[Major distinctions between the two oligopeptide permease systems of Bacillus subtilis with respect to signaling, development and evolutionary divergence](https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001382).\
<u>Microbiology</u>. 2023 Sep.

Gralka M, **Pollak, S**, Cordero OX.\
[Genome content predicts the carbon catabolic preferences of heterotrophic bacteria](https://www.nature.com/articles/s41564-023-01458-z).\
<u>Nature Microbiology</u>. 2023 Aug.

**Pollak, S**.\
[Plant DNA in feces as a nutritional crystal ball](https://www.pnas.org/doi/abs/10.1073/pnas.2309172120).\
<u>PNAS</u>. 2023 Jul. 

Szabo RE, Pontrelli S, Grilli J, Schwartzman JA, **Pollak, S**, Sauer U, Cordero OX.\
[Historical contingencies and phage induction diversify bacterioplankton communities at the microscale](https://www.pnas.org/doi/full/10.1073/pnas.2117748119).\
<u>PNAS</u>. 2022 Jul; 119(30):e2117748119

Pontrelli S, Szabo R, **Pollak, S**, Schwartzman J, Ledezma-Tejeida D, Cordero OX, Sauer U.\
[Metabolic cross-feeding structures the assembly of polysaccharide degrading communities](https://www.science.org/doi/full/10.1126/sciadv.abk3076).\
<u>Science Advances</u>. 2022 Feb. 8(8):eabk3076

**Pollak, S**, Gralka M, Sato Y, Schwartzman J, Lu, L, Cordero, OX.\
[Public good exploitation in natural bacterioplankton communities](https://www.science.org/doi/full/10.1126/sciadv.abi4717).\
<u>Science Advances</u>. 2021 Jul; 7(31):eabi4717

Liao J, Guo X, Weller DL, **Pollak, S**, Buckley DH, Wiedmann M, Cordero OX.\
[Nationwide genomic atlas of soil-dwelling Listeria reveals effects of selection and population ecology on pangenome evolution](https://www.nature.com/articles/s41564-021-00935-7).\
<u>Nature Microbiology</u>. 2021 Jul; 6:1021–1030

Ben-Zion I., **Pollak, S**., Eldar A.\
[Clonality and non-linearity drive facultative-cooperation allele diversity](https://www.nature.com/articles/s41396-018-0310-y).\
<u>The ISME journal</u>. 2019 Mar;13(3):824.

Bareia T., **Pollak, S**., Eldar A.\
[Self-sensing in Bacillus subtilis quorum-sensing systems](https://www.nature.com/articles/s41564-017-0044-z).\
<u>Nature Microbiology</u>. 2018 Jan; 3(1):83.

### Before 2018
Even-Tov E, Omer Bendori S, **Pollak, S**, Eldar A.\
[Transient Duplication-Dependent Divergence and Horizontal Transfer Underlie the Evolutionary Dynamics of Bacterial Cell-Cell Signaling](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2000330).\
<u>PLoS Biol</u>. 2016 Dec 29;14(12):e2000330.

Even-Tov E, Omer Bendori S, Valastyan J, Ke X, **Pollak, S**, Bareia T, Ben-Zion I, Bassler BL, Eldar A.\
[Social Evolution Selects for Redundancy in Bacterial Quorum Sensing](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002386).\
<u>PLoS Biol</u>. 2016 Feb 29;14(2):e1002386.

**Pollak, S**, Omer-Bendori S, Even-Tov E, Lipsman V, Bareia T, Ben-Zion I, Eldar A.\
[Facultative cheating supports the coexistence of diverse quorum-sensing alleles](https://www.pnas.org/doi/full/10.1073/pnas.1520615113).\
<u>PNAS</u>. 2016 Feb 23;113(8):2152-7.

 <p style="padding-left:50px;">
  <em>Commentary by Pérez-Escudero A., Gore J.</br>
  <a href="https://www.pnas.org/doi/full/10.1073/pnas.1600174113">Selection favors incompatible signaling in bacteria</a>.</br>
  <u>PNAS</u> 2016 February 23;113(8):1968-1970</em>
 </p>

**Pollak, S**, Omer Bendori S, Eldar A.\
[A complex path for domestication of B. subtilis sociality](https://link.springer.com/article/10.1007/s00294-015-0479-9).\
<u>Curr Genet</u>. 2015 Nov;61(4):493-6.

Bendori SO, **Pollak, S**, Hizi D, Eldar A.\
[The RapP-PhrP quorum sensing system of Bacillus subtilis strain NCIB3610 affect biofilm formation through multiple targets, due to an atypical signal-insensitive allele of RapP](https://journals.asm.org/doi/10.1128/JB.02382-14).\
<u>J Bacteriol</u>. 2014 Nov 24. pii: JB.02382-14.

Halimi Y, Dessau M, **Pollak, S**, Ast T, Erez T, Livnat-Levanon N, Karniol B, Hirsch JA, Chamovitz DA.\
[COP9 signalosome subunit 7 from Arabidopsis interacts with and regulates the small subunit of ribonucleotide reductase (RNR2)](https://link.springer.com/article/10.1007/s11103-011-9795-8).\
<u>Plant Mol Biol</u>. 2011 Sep;77(1-2):77-89.
