<!doctype html><html lang=en dir=ltr class=scroll-smooth data-default-appearance=light data-auto-appearance=false><head><meta charset=utf-8><meta http-equiv=content-language content="en"><meta name=viewport content="width=device-width,initial-scale=1"><meta name=theme-color content="rgb(255,255,255)"><meta http-equiv=X-UA-Compatible content="ie=edge"><title>&#183; Environmental Systems Microbiology @ Univie</title>
<meta name=title content=" &#183; Environmental Systems Microbiology @ Univie"><meta name=description content="Shaul Pollak Lab University of Vienna CMESS DOME about research project"><link rel=canonical href=/about/><link type=text/css rel=stylesheet href=/css/main.bundle.min.bbbd7a28e5cd8a7e81f4eb57c71ed0fe171a3663566e6e8f239bd31783b058f5.css integrity="sha256-u716KOXNin6B9OtXxx7Q/hcaNmNWbm6PI5vTF4OwWPU="><script type=text/javascript src=/js/appearance.min.022d0ebc3b46a335eb1c7ef79b7f2de143d7cd5156d433638592ef1ce5f8554e.js integrity="sha256-Ai0OvDtGozXrHH73m38t4UPXzVFW1DNjhZLvHOX4VU4="></script><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=manifest href=/site.webmanifest><meta name=google-site-verification content="2P33wvsvNTeuaricvESlGLKS5X68CLAv5aGAiSmAIIQ"><meta property="og:title" content><meta property="og:description" content="Shaul Pollak Lab University of Vienna CMESS DOME about research project"><meta property="og:type" content="article"><meta property="og:url" content="/about/"><meta property="article:section" content><meta name=twitter:card content="summary"><meta name=twitter:title content><meta name=twitter:description content="Shaul Pollak Lab University of Vienna CMESS DOME about research project"><script type=application/ld+json>[{"@context":"https://schema.org","@type":"Article","articleSection":"","name":"","headline":"","description":"Shaul Pollak Lab University of Vienna CMESS DOME about research project","abstract":"We study how the function and genetic variation of microbial communities emerge from the complexity of interactions they contain.","inLanguage":"en","url":"\/about\/","author":{"@type":"Person","name":""},"mainEntityOfPage":"true","wordCount":"305"}]</script><script async src="https://www.googletagmanager.com/gtag/js?id=G-X77QK0ZF3Q"></script><script>var doNotTrack=!1;if(!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-X77QK0ZF3Q",{anonymize_ip:!1})}</script></head><body class="flex flex-col h-screen px-6 m-auto text-lg leading-7 max-w-7xl bg-neutral text-neutral-900 dark:bg-neutral-800 dark:text-neutral sm:px-14 md:px-24 lg:px-32"><div id=the-top class="absolute flex self-center"><a class="px-3 py-1 text-sm -translate-y-8 rounded-b-lg bg-primary-200 focus:translate-y-0 dark:bg-neutral-600" href=#main-content><span class="font-bold pe-2 text-primary-600 dark:text-primary-400">&darr;</span>Skip to main content</a></div><header class="py-6 font-semibold text-neutral-900 dark:text-neutral print:hidden sm:py-10"><nav class="flex items-start justify-between sm:items-center"><div class="flex flex-row items-center"><a class="decoration-primary-500 hover:underline hover:decoration-2 hover:underline-offset-2" rel=me href=/>Environmental Systems Microbiology @ Univie</a></div><ul class="flex list-none flex-col ltr:text-right rtl:text-left sm:flex-row"><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/about/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">About</span></a></li><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/team/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">Team</span></a></li><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/publications/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">Publications</span></a></li></ul></nav></header><div class="relative flex flex-col grow"><main id=main-content class=grow><article class=max-w-full><header><h1 class="mt-0 text-4xl font-extrabold text-neutral-900 dark:text-neutral"></h1></header><section class="max-w-full mt-6 prose dark:prose-invert"><img align=right src=litter_cycle.svg width=500 style=padding-right:20px;padding-top:10px;margin-top:0;margin-bottom:5px><div style=text-align:left;font-size:1.5rem;font-weight:600;padding-left:0;padding-right:0;max-width:50rem;margin:0;color:#2563eb>We study how the function and genetic variation of microbial communities emerge from the complexity of interactions they contain.</div><p>Microbial communities (also called microbiomes) drive biogeochemical cycling, and are critical for human health and agriculture. A deeper understanding of microbiomes and their interaction with the environment is critical for both basic science and many applications such as sustainable agriculture, composting, and climate modeling.</p><p>Our primary source of information about natural microbiomes is DNA sequence data. This is because microbes are tiny and we can&rsquo;t see them with the naked eye, but sampling them is (relatively) easy and sequencing their DNA is cheap and fast. Unfortunately, most of environmental genetic material is uncharacterized, and predicting ecologically meaningful phenotypes (traits) from genomes is hard even for well-characterized microbes. Therefore, the deluge of environmental genomic data produced by the advent of cheap sequencing is still underutilized. We develop combined experimental and machine-learning approaches to go beyond this genomic knowledge gap to allow robust and accurate mapping from microbiome genomic information to community trait configurations. Borrowing ideas from ecological, evolutionary and genetic theory (basically epistasis), we also develop the theory to translate these trait configurations to quantitative community function such as CO<sub>2</sub> production rate.</p><img align=left src=vario.svg width=450 style=padding-right:20px;padding-top:0;margin-top:0;margin-bottom:10px><div style=text-align:left;font-size:1.5rem;font-weight:600;padding-left:0;padding-right:0;max-width:60rem;margin:0;color:#2563eb>As environmental bacterial geneticists, we are also extremely interested in how host-bacteria genetic interactions evolve in complex ecological settings.</div><p>Understanding the causes and consequences of genetic (co-)variation is a fundamental question in biology, and we pursue it happily in the lab and in the field. Our model system is the amazing tri-partite Variovorax-Arabidopsis-Microbiome interaction. Variovorax is a genetically tractable (i.e., we can do targeted genetic manipulations in its genome), highly prevalent soil bacterial genus that interacts strongly with plants and the surrounding microbiome. We study the consequence of genetic variation in Variovorax strains on plant growth as it interacts with different plant genotypes and microbiomes.</p></section><footer class=pt-8></footer></article></main><footer class="py-10 print:hidden"><div class="flex items-center justify-between"><div></div></div></footer></div></body></html>