<!doctype html><html lang=en dir=ltr class=scroll-smooth data-default-appearance=light data-auto-appearance=false><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=theme-color content="#FFFFFF"><title>&#183; Environmental Systems Microbiology @ Univie</title>
<meta name=title content=" &#183; Environmental Systems Microbiology @ Univie"><script type=text/javascript src=/js/appearance.min.8a082f81b27f3cb2ee528df0b0bdc39787034cf2cc34d4669fbc9977c929023c.js integrity="sha256-iggvgbJ/PLLuUo3wsL3Dl4cDTPLMNNRmn7yZd8kpAjw="></script><link type=text/css rel=stylesheet href=/css/main.bundle.min.42ff6bb08ef6313f648422f421d349088ac6f80d586a244f077a9078f7738b97.css integrity="sha256-Qv9rsI72MT9khCL0IdNJCIrG+A1YaiRPB3qQePdzi5c="><meta name=description content="
      Shaul Pollak Lab CMESS DOME University of Vienna Publications
    "><link rel=canonical href=/publications/><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=manifest href=/site.webmanifest><meta name=google-site-verification content="2P33wvsvNTeuaricvESlGLKS5X68CLAv5aGAiSmAIIQ"><meta property="og:url" content="/publications/"><meta property="og:site_name" content="Environmental Systems Microbiology @ Univie"><meta property="og:title" content="Environmental Systems Microbiology @ Univie"><meta property="og:description" content="Shaul Pollak Lab CMESS DOME University of Vienna Publications"><meta property="og:locale" content="en"><meta property="og:type" content="article"><meta name=twitter:card content="summary"><meta name=twitter:title content="Environmental Systems Microbiology @ Univie"><meta name=twitter:description content="Shaul Pollak Lab CMESS DOME University of Vienna Publications"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","articleSection":"","name":"","headline":"","description":"Shaul Pollak Lab CMESS DOME University of Vienna Publications","inLanguage":"en","url":"\/publications\/","author":{"@type":"Person","name":""},"mainEntityOfPage":"true","wordCount":"629"}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","item":"/","name":"","position":1},{"@type":"ListItem","name":"","position":2}]}</script><script async src="https://www.googletagmanager.com/gtag/js?id=G-X77QK0ZF3Q"></script><script>var dnt,doNotTrack=!1;if(!1&&(dnt=navigator.doNotTrack||window.doNotTrack||navigator.msDoNotTrack,doNotTrack=dnt=="1"||dnt=="yes"),!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-X77QK0ZF3Q")}</script></head><body class="m-auto flex h-screen max-w-7xl flex-col bg-neutral px-6 text-lg leading-7 text-neutral-900 dark:bg-neutral-800 dark:text-neutral sm:px-14 md:px-24 lg:px-32"><div id=the-top class="absolute flex self-center"><a class="-translate-y-8 rounded-b-lg bg-primary-200 px-3 py-1 text-sm focus:translate-y-0 dark:bg-neutral-600" href=#main-content><span class="pe-2 font-bold text-primary-600 dark:text-primary-400">&darr;</span>Skip to main content</a></div><header class="py-6 font-semibold text-neutral-900 dark:text-neutral sm:py-10 print:hidden"><nav class="flex items-start justify-between sm:items-center"><div class="flex flex-row items-center"><a class="decoration-primary-500 hover:underline hover:decoration-2 hover:underline-offset-2" rel=me href=/>Environmental Systems Microbiology @ Univie</a></div><ul class="flex list-none flex-col text-end sm:flex-row"><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/about/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">About</span></a></li><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/team/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">Team</span></a></li><li class="group mb-1 sm:mb-0 sm:me-7 sm:last:me-0.5"><a href=/publications/ title><span class="decoration-primary-500 group-hover:underline group-hover:decoration-2 group-hover:underline-offset-2">Publications</span></a></li></ul></nav></header><div class="relative flex grow flex-col"><main id=main-content class=grow><article class=max-w-full><header><h1 class="mt-0 text-4xl font-extrabold text-neutral-900 dark:text-neutral"></h1></header><section class="prose mt-6 max-w-full dark:prose-invert"><hr><p><b>Google Scholar profile: <a href="https://scholar.google.com/citations?user=jIHJECUAAAAJ&hl=en&oi=ao">Link</a></b></p><hr><h2 id=preprints class="relative group">Preprints <span class="absolute top-0 w-6 transition-opacity opacity-0 -start-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#preprints aria-label=Anchor>#</a></span></h2><p>Anna Lopatina, Mariusz Ferenc, Nina Bartlau, Michael Wolfram, Kerrin Steensen, Sebastiano Muscia, Fatima Hussain, Madeleine Schnurer, Leila Afjehi-Sadat, <strong>Shaul Pollak</strong>, Martin F Polz<br><a href=https://www.biorxiv.org/content/10.1101/2024.06.14.598830.abstract target=_blank rel=noreferrer>Interpretable machine learning reveals a diverse arsenal of anti-defenses in bacterial viruses</a><br><u>bioRxiv</u>. 2024 Jun.</p><p>Xiaoqian Annie Yu, Cameron R Strachan, Craig W Herbold, Michaela Lang, Christoph Gasche, Athanasios Makristathis, Nicola Segata, <strong>Shaul Pollak</strong>, Adrian Tett, Martin F Polz<br><a href=https://www.biorxiv.org/content/10.1101/2024.05.25.595854v1.abstract target=_blank rel=noreferrer>Genome-wide sweeps create fundamental ecological units in the human gut microbiome</a><br><u>bioRxiv</u>. 2024 May.</p><h2 id=published class="relative group">Published <span class="absolute top-0 w-6 transition-opacity opacity-0 -start-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#published aria-label=Anchor>#</a></span></h2><h3 id=2024 class="relative group">2024 <span class="absolute top-0 w-6 transition-opacity opacity-0 -start-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#2024 aria-label=Anchor>#</a></span></h3><p>Jamie W Becker, <strong>Shaul Pollak</strong>, Jessie W Berta-Thompson, Kevin W Becker, Rogier Braakman, Keven D Dooley, Thomas Hackl, Allison Coe, Aldo Arellano, Kristen N LeGault, Paul M Berube, Steven J Biller, Andrés Cubillos-Ruiz, Benjamin AS Van Mooy, Sallie W Chisholm
<a href=https://journals.asm.org/doi/abs/10.1128/mbio.03497-23 target=_blank rel=noreferrer>Novel isolates expand the physiological diversity of Prochlorococcus and illuminate its macroevolution</a>
<u>Mbio</u>. 2024 Nov.</p><p>Ksenia Guseva, Moritz Mohrlok, Lauren Alteio, Hannes Schmidt, <strong>Shaul Pollak</strong>, Christina Kaiser<br><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012320" target=_blank rel=noreferrer>Bacteria face trade-offs in the decomposition of complex biopolymers</a><br><u>PLOS Computational Biology</u>. 2024 Aug.</p><p>Xiaoyu Shan, Philip A Wasson, YuanQiao Rao, Scott Backer, Lyndsay Leal, Vurtice C Albright III, Wei Gao, Yunzhou Chai, Andreas Sichert, <strong>Shaul Pollak</strong>, Ligeng Yin, Otto X Cordero<br><a href=https://pubs.acs.org/doi/abs/10.1021/acs.est.4c02769 target=_blank rel=noreferrer>Simple genomic traits predict rates of polysaccharide biodegradation</a><br><u>Environmental Science & Technology</u>. 2024 Jul.</p><h3 id=2018-2023 class="relative group">2018-2023 <span class="absolute top-0 w-6 transition-opacity opacity-0 -start-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#2018-2023 aria-label=Anchor>#</a></span></h3><p>Bareia T., <strong>Pollak S.</strong>, Guler P., Puyesky S., Eldar A.<br><a href=https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001382 target=_blank rel=noreferrer>Major distinctions between the two oligopeptide permease systems of Bacillus subtilis with respect to signaling, development and evolutionary divergence</a>.<br><u>Microbiology</u>. 2023 Sep.</p><p>Gralka M, <strong>Pollak, S</strong>, Cordero OX.<br><a href=https://www.nature.com/articles/s41564-023-01458-z target=_blank rel=noreferrer>Genome content predicts the carbon catabolic preferences of heterotrophic bacteria</a>.<br><u>Nature Microbiology</u>. 2023 Aug.</p><p><strong>Pollak, S</strong>.<br><a href=https://www.pnas.org/doi/abs/10.1073/pnas.2309172120 target=_blank rel=noreferrer>Plant DNA in feces as a nutritional crystal ball</a>.<br><u>PNAS</u>. 2023 Jul.</p><p>Szabo RE, Pontrelli S, Grilli J, Schwartzman JA, <strong>Pollak, S</strong>, Sauer U, Cordero OX.<br><a href=https://www.pnas.org/doi/full/10.1073/pnas.2117748119 target=_blank rel=noreferrer>Historical contingencies and phage induction diversify bacterioplankton communities at the microscale</a>.<br><u>PNAS</u>. 2022 Jul; 119(30):e2117748119</p><p>Pontrelli S, Szabo R, <strong>Pollak, S</strong>, Schwartzman J, Ledezma-Tejeida D, Cordero OX, Sauer U.<br><a href=https://www.science.org/doi/full/10.1126/sciadv.abk3076 target=_blank rel=noreferrer>Metabolic cross-feeding structures the assembly of polysaccharide degrading communities</a>.<br><u>Science Advances</u>. 2022 Feb. 8(8):eabk3076</p><p><strong>Pollak, S</strong>, Gralka M, Sato Y, Schwartzman J, Lu, L, Cordero, OX.<br><a href=https://www.science.org/doi/full/10.1126/sciadv.abi4717 target=_blank rel=noreferrer>Public good exploitation in natural bacterioplankton communities</a>.<br><u>Science Advances</u>. 2021 Jul; 7(31):eabi4717</p><p>Liao J, Guo X, Weller DL, <strong>Pollak, S</strong>, Buckley DH, Wiedmann M, Cordero OX.<br><a href=https://www.nature.com/articles/s41564-021-00935-7 target=_blank rel=noreferrer>Nationwide genomic atlas of soil-dwelling Listeria reveals effects of selection and population ecology on pangenome evolution</a>.<br><u>Nature Microbiology</u>. 2021 Jul; 6:1021–1030</p><p>Ben-Zion I., <strong>Pollak, S</strong>., Eldar A.<br><a href=https://www.nature.com/articles/s41396-018-0310-y target=_blank rel=noreferrer>Clonality and non-linearity drive facultative-cooperation allele diversity</a>.<br><u>The ISME journal</u>. 2019 Mar;13(3):824.</p><p>Bareia T., <strong>Pollak, S</strong>., Eldar A.<br><a href=https://www.nature.com/articles/s41564-017-0044-z target=_blank rel=noreferrer>Self-sensing in Bacillus subtilis quorum-sensing systems</a>.<br><u>Nature Microbiology</u>. 2018 Jan; 3(1):83.</p><h3 id=before-2018 class="relative group">Before 2018 <span class="absolute top-0 w-6 transition-opacity opacity-0 -start-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#before-2018 aria-label=Anchor>#</a></span></h3><p>Even-Tov E, Omer Bendori S, <strong>Pollak, S</strong>, Eldar A.<br><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2000330" target=_blank rel=noreferrer>Transient Duplication-Dependent Divergence and Horizontal Transfer Underlie the Evolutionary Dynamics of Bacterial Cell-Cell Signaling</a>.<br><u>PLoS Biol</u>. 2016 Dec 29;14(12):e2000330.</p><p>Even-Tov E, Omer Bendori S, Valastyan J, Ke X, <strong>Pollak, S</strong>, Bareia T, Ben-Zion I, Bassler BL, Eldar A.<br><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002386" target=_blank rel=noreferrer>Social Evolution Selects for Redundancy in Bacterial Quorum Sensing</a>.<br><u>PLoS Biol</u>. 2016 Feb 29;14(2):e1002386.</p><p><strong>Pollak, S</strong>, Omer-Bendori S, Even-Tov E, Lipsman V, Bareia T, Ben-Zion I, Eldar A.<br><a href=https://www.pnas.org/doi/full/10.1073/pnas.1520615113 target=_blank rel=noreferrer>Facultative cheating supports the coexistence of diverse quorum-sensing alleles</a>.<br><u>PNAS</u>. 2016 Feb 23;113(8):2152-7.</p><p style=padding-left:50px><em>Commentary by Pérez-Escudero A., Gore J.</br><a href=https://www.pnas.org/doi/full/10.1073/pnas.1600174113>Selection favors incompatible signaling in bacteria</a>.</br><u>PNAS</u> 2016 February 23;113(8):1968-1970</em></p><p><strong>Pollak, S</strong>, Omer Bendori S, Eldar A.<br><a href=https://link.springer.com/article/10.1007/s00294-015-0479-9 target=_blank rel=noreferrer>A complex path for domestication of B. subtilis sociality</a>.<br><u>Curr Genet</u>. 2015 Nov;61(4):493-6.</p><p>Bendori SO, <strong>Pollak, S</strong>, Hizi D, Eldar A.<br><a href=https://journals.asm.org/doi/10.1128/JB.02382-14 target=_blank rel=noreferrer>The RapP-PhrP quorum sensing system of Bacillus subtilis strain NCIB3610 affect biofilm formation through multiple targets, due to an atypical signal-insensitive allele of RapP</a>.<br><u>J Bacteriol</u>. 2014 Nov 24. pii: JB.02382-14.</p><p>Halimi Y, Dessau M, <strong>Pollak, S</strong>, Ast T, Erez T, Livnat-Levanon N, Karniol B, Hirsch JA, Chamovitz DA.<br><a href=https://link.springer.com/article/10.1007/s11103-011-9795-8 target=_blank rel=noreferrer>COP9 signalosome subunit 7 from Arabidopsis interacts with and regulates the small subunit of ribonucleotide reductase (RNR2)</a>.<br><u>Plant Mol Biol</u>. 2011 Sep;77(1-2):77-89.</p></section><footer class=pt-8></footer></article></main><footer class="py-10 print:hidden"><div class="flex items-center justify-between"><div></div><div class="flex flex-row items-center"></div></div></footer></div></body></html>